guides/17_Migration_to_3_0.md

```diff
@@ -64,7 +64,10 @@ await meetingSession.audioVideo.startVideoInput(videoInputDeviceInfo.deviceId);
 In v3, you should call `stopVideoInput` to stop the video input stream.
 
 ```js
-// v3
+// Before
```

Comment:
Do we need to mention explicitly that `startVideoInput` API does not accept `null` as argument since it's a change of parameter type.

Reply:
Don't think its needed since we already add the before example.